pkgpanda/__init__.py

```diff
@@ -678,9 +712,22 @@ def symlink_all(src, dest):
 
         active_buildinfo_full = {}
 
+        dcos_service_configuration = self._get_dcos_configuration_template()
+
         # Building up the set of users
         sysusers = UserManagement(self.__manage_users, self.__add_users)
 
+        def _get_service_names(_dir):
+            service_files = list(map(os.path.basename,
+                                     glob.glob(_dir + "/dcos.target.wants/*.service") +
```

Comment:
Here we are being explicit with path and matching it **only** against `*.service` file. That design was carefully chosen over more wider use of glob, which, I agree can be tricky.

If there is test case which can fail this logic, please share, I will cover this, otherwise nothing to be done here IHMO.